titus-common/src/test/java/com/netflix/titus/common/util/loadshedding/tokenbucket/TokenBucketAdmissionControllerTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.common.util.loadshedding.tokenbucket;
+
+import java.util.Collections;
+
+import com.netflix.titus.common.runtime.TitusRuntime;
+import com.netflix.titus.common.runtime.TitusRuntimes;
+import com.netflix.titus.common.util.loadshedding.AdmissionControllerRequest;
+import com.netflix.titus.common.util.loadshedding.AdmissionControllerResponse;
+import org.junit.Test;
+
+import static com.netflix.titus.common.util.loadshedding.tokenbucket.TokenBucketTestConfigurations.DEFAULT_SHARED_CONFIGURATION;
+import static com.netflix.titus.common.util.loadshedding.tokenbucket.TokenBucketTestConfigurations.NOT_SHARED_CONFIGURATION;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TokenBucketAdmissionControllerTest {
```

Comment:
add a test for a single `callerId` (or `callerIdPattern`) with different configs for different endpoints